contribs/gmf/less/desktop.less

```diff
@@ -64,6 +64,84 @@ main {
     bottom: 0;
     left: 0;
   }
+
+  .footer {
+    @height: @line-height-computed + 2 * @padding-small-vertical;
+    position: absolute;
+    bottom:  -@height;
+    background-color: fade(@main-bg-color, 80%);
+    width: 100%;
+    /* cancel default navbar bottom margin */
+    margin-bottom: 0;
+    /* buttons or inputs in bar are supposed to be '-sm' */
+    transition: 0.2s ease-out all;
+    &.active {
+      bottom: 0;
+    }
+    > div {
+      display: inline-block;
+    }
+
+    button.map-info {
+      position: absolute;
+      /* button is supposed to be .btn-sm */
+      bottom: @line-height-computed + 2 * @padding-small-vertical;
+      background-color: fade(@main-bg-color, 80%);
+      border-bottom-left-radius: 0;
+      border-bottom-right-radius: 0;
+      right: @app-margin;
+      border-bottom: 0;
+      padding: @half-app-margin @app-margin;
+    }
+    .btn {
+      background-color: lighten(@main-bg-color, 20%);
+      padding: @half-app-margin 0;
+    }
+    .form-group{
+      margin-bottom: 0;
+    }
+  }
+
+  [ngeo-scaleselector] {
+    min-width: 12rem;
+    .dropdown-menu {
+      min-width: 12rem;
```

Comment:
That's true but only for dropdown with size > 160px (~=16rem here).
I don't know why but bootstrap defin a `min-width: 160px` on its `.dropdown-menu`.